pandas/core/series.py

```diff
@@ -173,12 +174,16 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
                 raise NotImplementedError("initializing a Series from a "
                                           "MultiIndex is not supported")
             elif isinstance(data, Index):
-                # need to copy to avoid aliasing issues
                 if name is None:
                     name = data.name
 
-                data = data._to_embed(keep_tz=True, dtype=dtype)
+                if dtype is not None:
+                    data = data.astype(dtype)
+
+                # need to copy to avoid aliasing issues
```

Comment:
only if you are not already astyping do you need to copy

Reply:
you can indeed do a `elif`, as the `astype` already copies by default even if the dtype match?

Reply:
FYI, this is just a comment I moved from up above to be closer to the copy. You think we can move it into the elif with the `astype`?

Reply:
yes I think you can move it (IOW if dtype is None), astype will copy (so if dtype is specified this would copy twice)